docs/Modders/uniques.md

```diff
@@ -3264,6 +3264,9 @@ Simple unique parameters are explained by mouseover. Complex parameters are expl
 ??? example  "&lt;when nuclear weapons are enabled&gt;"
 	Applicable to: Conditional
 
+??? example  "&lt;when nuclear weapons are disabled&gt;"
+	Applicable to: Conditional
+
```

Comment:
Don't need to rebuild the docs, as those should be built at release. Avoidiing the docs could save conflicts later on.